src/index.js

```diff
@@ -58,15 +58,26 @@ export default function ({types: t}) {
         }
       },
       'FunctionDeclaration|FunctionExpression|ArrowFunctionExpression'(path, state) {
-        if(!isStatelessComponent(path)) {
+        if (!isStatelessComponent(path)) {
           return;
         }
-        if(!path.parentPath.node.id) {
-          return;
+        const nodeType = path.node.type;
+
+        let className;
```

Comment:
can you assign `className = path.parentPath.node.id.name;` here and move the else statement from below up?

your statements are correct here, but I just want to prevent className being null in case this code gets refactored or changed.

Reply:
Sure will do 👍